File: old_enough.py
#AH 6th period Old Enough-Python

# ask how old they are
age = int(input("How old are you? "))

# check if they can vote
if age >= 18:
    print("You are old enough to vote!")
# check if they can drive
elif age >= 16:
    print("You are old enough to drive!")
# check if they can get a learner's permit
elif age >= 15:
    print("You are old enough to get a learners permit!")
# check if they can go to school
elif age >= 5:
    print("You are old enough to go to school!")
# check if they are too young
else:
    print("You’re too young for these, sorry.")

File: Old_Enough.c
//AH period 6 Old Enough - C

#include <stdio.h>

int main() {
    int age;
    printf("How old are you: ");
    scanf("%d", &age);

    // checking if the person is old enough 
    if (age >= 18) {
        printf("You are old enough to vote and drive!\n");
    } else if (age >= 16) {
        printf("You are old enough to drive!\n");
    } else if (age >= 15) {
        printf("You are old enough to get a learners permit!\n");
    } else if (age >= 5) {
        printf("You are old enough to go to school!\n");
    } else {
        printf("You are not old enough for any of these, sorry :(.\n");
    }

    return 0;
}
